tools/ALARAJOYWrapper/alarajoy_QA_notebook.ipynb

```diff
@@ -0,0 +1,85 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1112b209",
+   "metadata": {},
+   "source": [
+    "# **ALARAJOY Library Conversion QA Notebook**\n",
+    "\n",
+    "\n",
+    "This Jupyter Notebook is designed to enable comparisons for neutron activation responses of a given single parent element as calculated by ALARA for the purpose of validating ALARAJOY-processed data from the FENDL3.2x data sets."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3298730d",
+   "metadata": {},
+   "source": [
+    "**Import Packages**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "57313772",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import alarajoy_QA as qa\n",
+    "import importlib"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d37d40a3",
+   "metadata": {},
+   "source": [
+    "**Run ALARA with each prepared binary data library**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4eae0d85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "importlib.reload(qa)\n",
+    "data_soure = {\n",
+    "    'fendl2' : '/groupspace/shared/n/nukecode/ALARA/data/fendl2bin',\n",
+    "    'fendl3' : '../../examples/data/fendl3'\n",
+    "}\n",
+    "\n",
+    "element = input('Select single parent element to evaluate: ').lower()\n",
+    "\n",
+    "for libname, binary in data_soure.items():\n",
+    "    alara_input = qa.fill_alara_template(element, binary)\n",
+    "    qa.write_alara_input_file(alara_input)\n",
+    "    output = qa.run_alara(element,libname)\n",
+    "    data_soure[libname] = output"
```

Comment:
Do you intend to overwrite the `data_source` here?  Might make sense to store this in another dictionary.